pilot/pkg/bootstrap/server.go

```diff
@@ -823,6 +823,16 @@ func (s *Server) initEventHandlers() error {
 				ConfigTypesUpdated: map[resource.GroupVersionKind]struct{}{curr.GroupVersionKind(): {}},
 				Reason:             []model.TriggerReason{model.ConfigUpdate},
 			}
+
+			// Additional tracking for PeerAuthentication, which triggers EDS push.
```

Comment:
Does PeerAuthentication trigger inbound listener update?

Reply:
Yes. It should.

Reply:
yeah. it does.